ngraph.mk

```diff
@@ -82,6 +82,21 @@ NGRAPH_BUILD_DIR := $(ROOTDIR)/3rdparty/ngraph-mxnet-bridge/build
 NGRAPH_INSTALL_DIR := $(ROOTDIR)/3rdparty/ngraph-mxnet-bridge/build
 MXNET_LIB_DIR := $(ROOTDIR)/lib
 
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Linux)
+	OS := $(shell cat /etc/os-release)
+	ifeq ($(findstring CentOS,$(OS)), CentOS)
+		OS := "CentOS"
+	else
+		OS := "Ubuntu"
+	endif
+endif
+
+ifeq ($(OS), "CentOS")
+	NGRAPH_LIB_DIR := $(NGRAPH_INSTALL_DIR)/lib64
+else
+	NGRAPH_LIB_DIR := $(NGRAPH_INSTALL_DIR)/lib
+endif
```

Comment:
This code assumes that any Linux computer on which this runs is either CentOS or Ubuntu.

Even if the code will happen to work okay on other major Linux's (RHEL, SuSE, etc.), it's a bit confusing to have the value of `OS` be `"Ubuntu"` for those other distributions.

Would it make sense to actually handle three cases?  I.e., Ubuntu, CentOS, and "other"?